packages/why3/why3.1.4.0/opam

```diff
@@ -0,0 +1,74 @@
+opam-version: "2.0"
+maintainer: "guillaume.melquiond@inria.fr"
+authors: [
+  "François Bobot"
+  "Jean-Christophe Filliâtre"
+  "Claude Marché"
+  "Guillaume Melquiond"
+  "Andrei Paskevich"
+]
+
+homepage: "http://why3.lri.fr/"
+license: "LGPL-2.1-only"
+doc: "http://why3.lri.fr/doc/"
+bug-reports: "https://gitlab.inria.fr/why3/why3/issues"
+dev-repo: "git+https://gitlab.inria.fr/why3/why3.git"
+
+tags: [
+  "deductive"
+  "program verification"
+  "formal specification"
+  "automated theorem prover"
+  "interactive theorem prover"
+]
+
+build: [
+  ["./autogen.sh"] {dev} # when pinning, there might be no configure file
+  ["./configure"
+   "--prefix" prefix
+   "--disable-frama-c"
+   "--disable-coq-libs"
+   "--disable-js-of-ocaml"
+   "--disable-ide"]
+  [make "-j%{jobs}%" "all" "opt" "byte"]
+  [make "doc" "stdlibdoc" "apidoc"] {with-doc}
+]
+
+install: [
+  [make "install" "install-lib"]
+  [make "DOCDIR=%{_:doc}%" "install-doc"] {with-doc}
+]
+
+depends: [
+  "conf-autoconf" {build & dev}
+  "ocaml" {>= "4.05.0"}
+  "ocamlfind" {build}
+  "menhir" {>= "20151112"}
+  "num"
+]
+
+depopts: [
+  "zarith"
+  "camlzip"
+  "ocamlgraph"
```

Comment:
with ocamlgraph installed this package fails with
```
# Linking  lib/plugins/genequlin.cmo
# Linking  lib/plugins/dimacs.cmo
# Linking  bin/why3extract.cma
# Linking  bin/why3wc.cma
# Linking  bin/why3webserver.cma
# Linking  bin/why3shell.cma
# Linking  bin/isabelle_client.byte
# Linking  bin/why3pp.cma
# Ocamlopt src/driver/collect_data_model.ml
# Linking  bin/why3prove.cma
# Linking  lib/plugins/hypothesis_selection.cmo
# File "_none_", line 1:
# Error: File graph.cmo not found
# make: *** [Makefile:587: lib/plugins/hypothesis_selection.cmo] Error 2
# make: *** Waiting for unfinished jobs....
```

Reply:
The packaging of ocamlgraph changed between 1.8 and 2.0. I have disabled support for ocamlgraph 2.

Reply:
This now passes CI.